components/brave_wallet_ui/page/screens/swap/components/swap/quote-info/quote-info.tsx

```diff
@@ -421,14 +434,27 @@ export const QuoteInfo = (props: Props) => {
       >
         {!showAdvancedInformation && selectedQuoteOption && (
           <Row justifyContent='space-between'>
-            <Row width='unset'>
-              <LPIcon icon={LPMetadata[selectedQuoteOption.provider]} />
+            <Row
+              width='unset'
+              gap='8px'
+            >
+              {firstRoutesIcon ? (
+                <LiquidityProviderIcon
+                  icon={firstRoutesIcon}
+                  size='16px'
+                />
+              ) : (
+                <BankIcon size='16px' />
+              )}
               <Text
                 textSize='12px'
                 isBold={true}
                 textColor='primary'
               >
-                {selectedQuoteOption.provider}
+                {firstRoutesName}{' '}
+                {additionalRoutesLength !== 0
+                  ? `+ ${additionalRoutesLength}`
+                  : ''}
```

Comment:
@Douglashdaniel `Paraswap + 2 routes` should instead be called `Paraswap + 2 steps`.

Nice to have: we may have a single additional step, hence it'll be `Paraswap + 1 step`.